cloudformation/application-setup-basic/stacks/application/service.yml

```diff
@@ -106,6 +111,18 @@ Resources:
               awslogs-group: !Ref 'ServiceName'
               awslogs-region: !Ref AWS::Region
               awslogs-stream-prefix: !Ref 'ServiceName'
+          Environment:
```

Comment:
This is the place where we can inject env vars into the Docker container. For the user pool ID and the client ID, we simply import the values that the cognito stack exports. For the secret, this doesn't work, because CloudFormation doesn't allow to export it.